### PR TITLE
feat(query): added "GKE Using Default Service Account" for Ansible and Terraform

### PR DESCRIPTION
**Pull request by cosmicgirl97**
**Proposed Changes**
- Added "GKE Using Default Service Account" for Ansible and Terraform

🚨 SEVERITY AND CATEGORY NEED REVIEW 🚨

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

**Review by JoaoDuarteGomes — APPROVED** [review]
LGTM.

File: assets/queries/ansible/gcp/gke_using_default_service_account/query.rego [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	ansLib.checkState(container_cluster)
+
+	not common_lib.valid_key(container_cluster.node_config, "service_account")
+
+	result := {
+		"documentId": id,
+		"searchKey": sprintf("name={{%s}}.{{%s}}.node_config", [task.name, modules[m]]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "'service_account' is not default",
+		"keyActualValue": "'service_account' is default",
```
Comment (joaoReigota1) [inline assets/queries/ansible/gcp/gke_using_default_service_account/query.rego:20]: Key expected/actual value should indicate that service_account is missing

**Review by joaoReigota1 — CHANGES_REQUESTED** [review]

**Review by JoaoDuarteGomes — APPROVED** [review]
LGTM.

**Review by joaoReigota1 — APPROVED** [review]
LGTM

**Pull request merged**